### PR TITLE
Small snforge docs fixes

### PR DESCRIPTION
**Pull request by drknzz**
## Introduced changes

- Small snforge docs changes

## Checklist

- [x] Linked relevant issue
- [x] Updated relevant documentation
- [x] Added relevant tests
- [x] Performed self-review of the code
- [x] Added changes to `CHANGELOG.md`

### CODE REVIEW

File: docs/src/testing/contracts.md [hunk 1]
```diff
@@ -105,7 +116,9 @@ If we called this function in a test, it would result in a failure.
 ```rust
 #[test]
 fn failing() {
-    // ...
+    let contract_address = deploy_contract('HelloStarknet');
```
Comment (piotmag769) [inline docs/src/testing/contracts.md:119]: There is not such library function as `deploy_contract` yet, it is usually a helper function I write myself xD

Comment (drknzz) [reply docs/src/testing/contracts.md:119]: Maybe it would be worth to have such helper function in the docs?

Comment (drknzz) [reply docs/src/testing/contracts.md:119]: I mean here, in the guide, where users tend to follow along.

Comment (piotmag769) [reply docs/src/testing/contracts.md:119]: I think it is enough to write

```cairo
// ...
let contract_address = deploy(prepared);
// the rest
```

File: docs/src/testing/contracts.md [hunk 2]
```diff
@@ -130,7 +145,7 @@ Using `SafeDispatcher` we can test that the function in fact panics with an expe
 ```rust
 #[test]
 fn handling_errors() {
-    // ...
+    let contract_address = deploy(prepared).unwrap();
```
Comment (piotmag769) [inline docs/src/testing/contracts.md:148]: Missing `// ...`

**Pull request merged**